modules/BookCollectClass.js

```diff
@@ -0,0 +1,20 @@
+export default class BookCollect {
+  constructor() {
+    this.books = [];
+  }
+
+  removeBook(index) {
+    this.books = this.books.filter((item, i) => i !== index);
+    localStorage.setItem('Books', JSON.stringify(this.books));
+    window.location.reload();
+  }
+
+  addBook(BookName, Author) {
+    const book = {
+      name: BookName,
+      author: Author,
+    };
+    this.books.push(book);
+    localStorage.setItem('Books', JSON.stringify(this.books));
+  }
```

Comment:
- [ ] The point of this milestone is to modularize your code as much as possible. You could achieve more of this by creating a separate module for saving to localStorage. You can then import the module and use the function here and for the `addBook` function as well. Kindly do so